bicep/network.bicep

```diff
@@ -160,10 +163,33 @@ var aks_baseSubnet =  {
     }: {})
 }
 
+var aks_podSubnet_name = 'aks-sn-pods'
```

Comment:
`-sn` is the subnet suffix used for the other subnets.

for consistency we should probably change the pod subnet to `aks-pods-sn`

![image](https://user-images.githubusercontent.com/17914476/235670654-e00561fa-0b28-48e3-9a73-4de8eda7e391.png)

Reply:
Good feedback! All done, thanks. :)